.github/workflows/emoji-data-tests.yml

```diff
@@ -0,0 +1,68 @@
+name: Emoji data
```

Comment:
can we do without a dedicated action?
I prefer centralizing all components in one to ease maintenance